src/frontend/src/components/CollapsibleInstructions/style.ts

```diff
@@ -30,6 +29,37 @@ export const HeaderWrapper = styled("div", {
   }}
 `;
 
+export const InstructionsTitle = styled("div", {
+  shouldForwardProp: (prop) => !doNotForwardProps.includes(prop as string),
+})`
+  ${(props) => {
+    const spaces = getSpaces(props);
+    return `
+    margin-bottom: ${spaces?.xxs}px;
+  `;
+  }}
+  ${(props: HeaderProps) => {
+    const { headerSize } = props;
+    return fontHeader(headerSize);
+  }}
+`;
+
+export const SecondInstructionsTitle = styled("div", {
```

Comment:
This looks almost identical to `InstructionsTitle`. If they should stay in sync with each other, I wonder if It would be good to have this extend that definition to keep it DRY?

Reply:
yep made some changes